src/Microsoft.AspNet.Authentication.Cookies/CookieAuthenticationHandler.cs

```diff
@@ -279,6 +279,7 @@ private CookieOptions BuildCookieOptions()
             var context = new CookieSigningOutContext(
                 Context,
                 Options,
+                new AuthenticationProperties(signOutContext.Properties),
```

Comment:
Required params go in the constructor (especially if they're passed to a base constructor). optional params use properties.